Shared/Samples/Show service area/ShowServiceAreaView.swift

```diff
@@ -48,56 +53,79 @@ struct ShowServiceAreaView: View {
-                                Stepper("First: \(firstTimeBreak)", value: $firstTimeBreak, in: 1...15)
-                                Stepper("Second: \(secondTimeBreak)", value: $secondTimeBreak, in: 1...15)
-                            }
-                            .navigationTitle("Time Breaks")
-                            .navigationBarTitleDisplayMode(.inline)
-                            .toolbar {
-                                ToolbarItem(placement: .confirmationAction) {
-                                    Button("Done") { settingsArePresented = false }
-                                }
-                            }
-                        }
-                        .presentationDetents([.fraction(0.25)])
-                        .frame(idealWidth: 320, idealHeight: 160)
+                    .popover(isPresented: $showSettings) {
+                        settingsPopover
+                            .frame(idealWidth: 400, idealHeight: 400)
+                            .presentationCompactAdaptation(.popover)
                     }
+                    
                     Spacer()
-                    Button("Service Area") {
+                    Button("Show Service Area") {
                         Task {
                             isSolvingServiceArea = true
                             do {
                                 try await model.showServiceArea(
-                                    timeBreaks: [Double(firstTimeBreak), Double(secondTimeBreak)]
+                                    timeBreaks: [
+                                        Double(firstTimeBreak),
+                                        Double(secondTimeBreak)
+                                    ]
                                 )
                             } catch {
                                 self.error = error
                             }
                             isSolvingServiceArea = false
                         }
                     }
-                    .disabled(isSolvingServiceArea)
+                    .disabled(!isServiceAreaSet || isSolvingServiceArea)
                     Spacer()
                     Button("Clear", systemImage: "trash") {
+                        isServiceAreaSet = false
                         model.removeAllGraphics()
                     }
+                    .disabled(!isServiceAreaSet)
                 }
             }
             .errorAlert(presentingError: $error)
     }
+    
+    @ViewBuilder var settingsPopover: some View {
+        NavigationStack {
+            Form {
+                Section("Facility Mode") {
+                    Picker("Mode", selection: $selectedGraphicType) {
+                        ForEach(GraphicType.allCases, id: \.self) {
+                            Text($0.label)
+                        }
+                    }
+                    .pickerStyle(.segmented)
+                }
+                Section("Service Time Radius") {
```

Comment:
Service Time Radius also seems a bit vague without context, hoping to find something better before this is merged.